wallet/common/symbol/src/utils/transaction-from-dto.js

```diff
@@ -0,0 +1,635 @@
+	const baseTransaction = baseTransactionFromDTO(transactionDTO, config);
+
+	switch (transactionDTO.transaction.type) {
+	case TransactionType.AGGREGATE_BONDED:
+	case TransactionType.AGGREGATE_COMPLETE:
+		return aggregateTransactionFromDTO(transactionDTO, config);
+	case TransactionType.TRANSFER:
+		return transferTransactionFromDTO(transactionDTO, config);
+	case TransactionType.ADDRESS_ALIAS:
+		return addressAliasTransactionFromDTO(transactionDTO, config);
+	case TransactionType.MOSAIC_ALIAS:
+		return mosaicAliasTransactionFromDTO(transactionDTO, config);
+	case TransactionType.NAMESPACE_REGISTRATION:
+		return namespaceRegistrationTransactionFromDTO(transactionDTO, config);
+	case TransactionType.MOSAIC_DEFINITION:
+		return mosaicDefinitionTransactionFromDTO(transactionDTO, config);
+	case TransactionType.MOSAIC_SUPPLY_CHANGE:
+		return mosaicSupplyChangeTransactionFromDTO(transactionDTO, config);
+	case TransactionType.MOSAIC_SUPPLY_REVOCATION:
+		return mosaicSupplyRevocationTransactionFromDTO(transactionDTO, config);
+	case TransactionType.SECRET_LOCK:
+		return secretLockTransactionFromDTO(transactionDTO, config);
+	case TransactionType.HASH_LOCK:
+		return hashLockTransactionFromDTO(transactionDTO, config);
+	case TransactionType.SECRET_PROOF:
+		return secretProofTransactionFromDTO(transactionDTO, config);
+	case TransactionType.VRF_KEY_LINK:
+		return vrfKeyLinkTransactionFromDTO(transactionDTO, config);
+	case TransactionType.ACCOUNT_KEY_LINK:
+		return accountKeyLinkTransactionFromDTO(transactionDTO, config);
+	case TransactionType.NODE_KEY_LINK:
+		return nodeKeyLinkTransactionFromDTO(transactionDTO, config);
+	case TransactionType.VOTING_KEY_LINK:
+		return votingKeyLinkTransactionFromDTO(transactionDTO, config);
+	case TransactionType.MOSAIC_GLOBAL_RESTRICTION:
+		return mosaicGlobalRestrictionTransactionFromDTO(transactionDTO, config);
+	case TransactionType.MOSAIC_ADDRESS_RESTRICTION:
+		return mosaicAddressRestrictionTransactionFromDTO(transactionDTO, config);
+	case TransactionType.ACCOUNT_OPERATION_RESTRICTION:
+		return accountOperationRestrictionTransactionFromDTO(transactionDTO, config);
+	case TransactionType.ACCOUNT_ADDRESS_RESTRICTION:
+		return accountAddressRestrictionTransactionFromDTO(transactionDTO, config);
+	case TransactionType.ACCOUNT_MOSAIC_RESTRICTION:
+		return accountMosaicRestrictionTransactionFromDTO(transactionDTO, config);
+	case TransactionType.MULTISIG_ACCOUNT_MODIFICATION:
+		return multisigAccountModificationTransactionFromDTO(transactionDTO, config);
+	case TransactionType.ACCOUNT_METADATA:
+		return accountMetadataTransactionFromDTO(transactionDTO, config);
+	case TransactionType.NAMESPACE_METADATA:
+		return namespaceMetadataTransactionFromDTO(transactionDTO, config);
+	case TransactionType.MOSAIC_METADATA:
+		return mosaicMetadataTransactionFromDTO(transactionDTO, config);
+	}
+
+	return baseTransaction;
+};
+
+export const baseTransactionFromDTO = (transactionDTO, config) => {
```

Comment:
I am not sure if this is a generated file, but I would assume export functions should have docs?

Reply:
Removed unused exports and added docs to remaining ones.